framework_tvb/tvb/tests/framework/adapters/simulator/simulator_adapter_test.py

```diff
@@ -112,38 +116,34 @@ def test_happy_flow_launch(self, connectivity_factory, operation_factory):
 
         SIMULATOR_PARAMETERS['connectivity'] = self.connectivity.gid
 
-        OperationService().initiate_prelaunch(self.operation, self.simulator_adapter, **SIMULATOR_PARAMETERS)
+        OperationService().initiate_prelaunch(self.operation, self.simulator_adapter, self.simulator_with_connectivity_model())
         sim_result = dao.get_generic_entity(TimeSeriesRegion, 'TimeSeriesRegion', 'type')[0]
         assert sim_result.read_data_shape() == (32, 1, self.CONNECTIVITY_NODES, 1)
 
-    def _estimate_hdd(self, new_parameters_dict):
-        """ Private method, to return HDD estimation for a given set of input parameters"""
-        filtered_params = self.simulator_adapter.prepare_ui_inputs(new_parameters_dict)
-        self.simulator_adapter.configure(**filtered_params)
-        return self.simulator_adapter.get_required_disk_size(**filtered_params)
+    def simulator_with_connectivity_model(self):
+        simulator_adapter_model = self.simulator_adapter.get_view_model()()
+
+        zip_path = path.join(path.dirname(tvb_data.__file__), 'connectivity', 'connectivity_96.zip')
+        TestFactory.import_zip_connectivity(self.test_user, self.test_project, zip_path, 'John Doe')
+        simulator_adapter_model.connectivity = TestFactory.get_entity(self.test_project, ConnectivityIndex).gid
+
+        return simulator_adapter_model
+
+    def _estimate_hdd(self):
+        """ Private method, to return HDD estimation for a given a model"""
+
+        self.simulator_adapter.configure(self.simulator_with_connectivity_model())
+        return self.simulator_adapter.get_required_disk_size(self.simulator_with_connectivity_model())
 
     def test_estimate_hdd(self, connectivity_factory):
         """
         Test that occupied HDD estimation for simulation results considers simulation length.
         """
-        factor = 5
-        simulation_parameters = copy(SIMULATOR_PARAMETERS)
-        ## Estimate HDD with default simulation parameters
-        estimate1 = self._estimate_hdd(simulation_parameters)
-        assert estimate1 > 1
+        self.test_user = TestFactory.create_user("Simulator_Adapter_User")
+        self.test_project = TestFactory.create_project(self.test_user, "Simulator_Adapter_Project")
```

Comment:
Why create the test_user and test_project in this unit-test? 
They do not seem used here